packages/opentelemetry-exporter-prometheus/src/PrometheusSerializer.ts

```diff
@@ -111,17 +111,19 @@ function stringify(
   let labelsStr = '';
 
   for (const [key, val] of Object.entries(labels)) {
+    const santisedLabelName = sanitizePrometheusMetricName(key);
```

Comment:
```suggestion
    const sanitisedLabelName = sanitisePrometheusMetricName(key);
```

Reply:
I have renamed it to use `z` to keep it better match the rest of the American spelling

Reply:
you still have typo, `i`